drivers/hmis/app/models/hmis/project_auto_exit_config.rb

```diff
@@ -5,6 +5,37 @@
 ###
 
 class Hmis::ProjectAutoExitConfig < Hmis::ProjectConfig
-  # This class is not yet used, but will be replacing AutoExitConfig
-  # TODO: Validate that config_options is not null and length_of_absence_days is defined on that json blob
+  validate :length_of_absence_days_ge_30
+
+  def length_of_absence_days=(value)
+    self.config_options = { 'length_of_absence_days': value }.to_json
+  end
+
+  def length_of_absence_days
+    JSON.parse(config_options)['length_of_absence_days']
+  end
+
+  def length_of_absence_days_ge_30
```

Comment:
Does this error-checking look sufficiently rails-idiomatic to you? I would welcome feedback and improvements on style!!

Reply:
Added some comments below 😃